content/components/sensor/dallas_temp.md

```diff
@@ -18,13 +18,16 @@ required to be set up in your configuration for this sensor to work.
 sensor:
   - platform: dallas_temp
     address: 0x1234567812345628
+    index: 1
```

Comment:
Move this to a second example or add a comment that only one of these can be used at a time.